data/sql/updates/pending_db_world/rev_1667732623534207400.sql

```diff
@@ -0,0 +1,20 @@
+-- Fire brigade esES / esMX
+INSERT INTO quest_offer_reward_locale (ID, locale, RewardText, VerifiedBuild) VALUES
+	(11360, 'esES', '¡Estás mojado y lleno de ceniza! ¡Tienes que haber estado apagando fuegos!$b$b¡Bien hecho! Como miembro honorífico de la brigada de bomberos, te ruego que regreses si alguna vez te necesitamos.', 0),
+	(11439, 'esES', '¡Estás mojado y lleno de ceniza! ¡Tienes que haber estado apagando fuegos!$b$b¡Bien hecho! Como miembro honorífico de la brigada de bomberos, te ruego que regreses si alguna vez te necesitamos.', 0),
+	(11440, 'esES', '¡Estás mojado y lleno de ceniza! ¡Tienes que haber estado apagando fuegos!$b$b¡Bien hecho! Como miembro honorífico de la brigada de bomberos, te ruego que regreses si alguna vez te necesitamos.', 0),
+    (11360, 'esMX', '¡Estás mojado y lleno de ceniza! ¡Tienes que haber estado apagando fuegos!$b$b¡Bien hecho! Como miembro honorífico de la brigada de bomberos, te ruego que regreses si alguna vez te necesitamos.', 0),
+	(11439, 'esMX', '¡Estás mojado y lleno de ceniza! ¡Tienes que haber estado apagando fuegos!$b$b¡Bien hecho! Como miembro honorífico de la brigada de bomberos, te ruego que regreses si alguna vez te necesitamos.', 0),
+	(11440, 'esMX', '¡Estás mojado y lleno de ceniza! ¡Tienes que haber estado apagando fuegos!$b$b¡Bien hecho! Como miembro honorífico de la brigada de bomberos, te ruego que regreses si alguna vez te necesitamos.', 0);
+	
+	UPDATE quest_template_locale SET ObjectiveText1 = 'Apaga fuegos' WHERE id = 11360 AND (locale = 'esES' OR locale = 'esMX');
```

Comment:
Backticks


Better syntax
```suggestion
	UPDATE `quest_template_locale` SET `ObjectiveText1` = 'Apaga fuegos' WHERE `id` = 11360 AND `locale` IN ('esES', 'esMX');
```